### PR TITLE
Update check action

### PR DESCRIPTION
**Pull request by hadley**

### CODE REVIEW

**Comment by hadley** [comment]
@MichaelChirico do you like the comment giving the code coverage report? (as opposed to the report included with the R CMD check results)? If you don't like it, I can also suppress it in this PR.

File: .github/workflows/R-CMD-check.yaml [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
-      - master
+    branches: [main, master]
 
 name: R-CMD-check
 
 jobs:
   R-CMD-check:
     runs-on: ${{ matrix.config.os }}
-    # accidentally committed an infinite loop that ran
-    #   for 6 hours in #208, so set a reasonable conservative upper bound.
```
Comment (MichaelChirico) [inline .github/workflows/R-CMD-check.yaml:17]: restore this?

**Comment by MichaelChirico** [comment]
> do you like the comment giving the code coverage report? (as opposed to the report included with the R CMD check results)?

sorry I'm just waking up and don't quite follow... this is about the comment codecov not posts to the PR with the results? I don't think I understand the alternative

**Comment by hadley** [comment]
In this thread you can see the codecov result twice: once as a comment (https://github.com/MichaelChirico/potools/pull/251#issuecomment-962609760) and once under "show all checks". Do you  want to see it only under "show all checks"?

File: .github/workflows/R-CMD-check.yaml (hunk 1 above)
Comment (hadley) [reply .github/workflows/R-CMD-check.yaml:17]: Sure, sorry.

**Comment by MichaelChirico** [comment]
gotcha... I'll have to adjust my workflow but let's give it a try

**Comment by MichaelChirico** [comment]
thanks!!!

**Pull request merged**